index.html

```diff
@@ -144,7 +172,8 @@ <h1>
                 <span>Skills</span>
                 <span>&#10095;</span>
             </div>
-        </section>
+        </div>
+    </div>
     </footer>
     <section class="form_section">
```

Comment:
Congratulations on your **contact section for desktop**! I have a couple of suggestions to help make **it resemble the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122) more closely**:

- [ ] I noticed that you're not using the correct `background image` for the contact section in your desktop version. Please **make sure to update it accordingly to align with the provided template.** 🙏

- [ ] Kindly check the `width` of your inputs as they appear smaller than in the given template. Adjusting their `width` **will help achieve a more accurate representation of the desired design.** 👌

    **_These small improvements will contribute to a more cohesive and visually appealing contact section. Thank you for considering these suggestions!_**
    
    ### Your design:
    ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/3101381a-7e2e-4b2e-9861-1992fd9285a7)

   ### Figma design:
   ![image](https://github.com/MustafaSamiShirzad/portfoliowebsite2/assets/107952724/c2961984-f86b-49c0-bd02-a3a7f6f02fdb)

Reply:
Resolved, thanks for your support